pandas/core/ops/__init__.py

```diff
@@ -818,6 +818,11 @@ def wrapper(self, other):
             # Defer to DataFrame implementation; fail early
             return NotImplemented
 
+        elif should_extension_dispatch(self, other):
+            # e.g. SparseArray
+            res_values = dispatch_to_extension_op(op, self, other)
+            return _construct_result(self, res_values, index=self.index, name=res_name)
```

Comment:
Is res_values always an array, or can it be a tuple of results, like from divmod?

Reply:
in this context its always an array